### PR TITLE
converting module data to json

### PR DESCRIPTION
**Pull request by Frijol**

### CODE REVIEW

**Comment by tcr** [comment]
This looks good, what else are you going to do with the data? @Frijol

**Comment by Frijol** [comment]
This should be used to generate a better/more useful modules page than we currently have.
This should also be the way people add community modules.

**Comment by Frijol** [comment]
Can also be used to whitelist USB modules

**Comment by Frijol** [comment]
ok @tcr data converted. Help fill out the USB section?

File: modules.json [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+  [
+    {
+      "name": "accelerometer",
+      "package": "accel-mma84",
+      "chip": "MMA8452Q",
+      "datasheet": "http://www.freescale.com/files/sensors/doc/data_sheet/MMA8452Q.pdf",
+      "image": "https://s3.amazonaws.com/technicalmachine-assets/product+pics/2014+05+15+production+modules/accelerometer.jpg",
+      "buy": "https://shop.tessel.io/Modules/Accelerometer%20Module",
+      "description": "Detect orientation and movement of your Tessel by measuring gravity / acceleration. 3-axis digital accelerometer; 12-bit resolution; selectable ±2g/±4g/±8g scales",
+      "supported": "Tessel 1, Tessel 2"
```
Comment (rwaldron) [inline modules.json]: Recommend: 

``` js
"supported": [ "Tessel 1", "Tessel 2"]
```

File: modules.json [hunk 2]
```diff
@@ -0,0 +1,486 @@
+{
+  "tenpin":
+  [
+    {
+      "name": "accelerometer",
+      "package": "accel-mma84",
+      "chip": "MMA8452Q",
+      "datasheet": "http://www.freescale.com/files/sensors/doc/data_sheet/MMA8452Q.pdf",
+      "image": "https://s3.amazonaws.com/technicalmachine-assets/product+pics/2014+05+15+production+modules/accelerometer.jpg",
```
Comment (rwaldron) [inline modules.json:9]: You may want to have more than one image for a given module, with meta data about each.

Recommend: 

``` js
"images": [ 
  { "description": "Front view", "src": "https://s3.amazonaws.com/technicalmachine-assets/product+pics/2014+05+15+production+modules/accelerometer.jpg" },
  { "description": "Rear view", "src": "rear.jpg" }
]
```

File: modules.json [hunk 3]
```diff
@@ -0,0 +1,486 @@
...
+      "buy": "https://shop.tessel.io/Modules/Accelerometer%20Module",
+      "description": "Detect orientation and movement of your Tessel by measuring gravity / acceleration. 3-axis digital accelerometer; 12-bit resolution; selectable ±2g/±4g/±8g scales",
+      "supported": "Tessel 1, Tessel 2"
+    },
```
Comment (rwaldron) [inline modules.json:17]: New key/value: 

``` js
"breadboard": {
  "png": "rendered-as-a.png",
  "svg": "rendered-as-a.svg",
  "fzz": "rendered-as-a.fzz"
},
```

File: modules.json (hunk 2 above)
Comment (Frijol) [inline modules.json:9]: Interesting, but I don't think it's going to come up; we don't have them for the existing tenpin modules, and we're not planning to add new ones soon.

File: modules.json (hunk 1 above)
Comment (Frijol) [inline modules.json]: Good call, will do

File: modules.json (hunk 3 above)
Comment (Frijol) [inline modules.json:17]: I don't have these, but you're welcome to add this if you have them.

**Comment by Frijol** [comment]
I think this is ready to merge (at least as a first pass)

File: modules.json (hunk 2 above)
Comment (tcr3dr) [inline modules.json:9]: I agree with Rick here, unless we don't changing the format later (and it probably won't be an issue)

**Comment by tcr3dr** [comment]
lgtm, rogue

**Pull request merged**